gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
-#Tue Feb 21 21:40:09 BRT 2023
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionSha256Sum=a62c5f99585dd9e1f95dab7b9415a0e698fa9dd1e6c38537faa81ac078f4d23e
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1-bin.zip
```

Comment:
Since you updated the APG to 8.0.0, the minimum required Gradle version is now 8.0, any reason to use 8.1?

Reply:
Yes, more than 100 bug fixes.

Reply:
Very good